index.html

```diff
@@ -57,71 +56,127 @@
 					        	<a class="nav-link" href="#">חברות הקבוצה</a>
 					        </li>
 					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">אחריות תאגידית
-					        	</a>
+					        	<a class="nav-link" href="#">אחריות תאגידית</a>
 					        </li>
 					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">אודות
-					        	</a>
+					        	<a class="nav-link" href="#">אודות</a>
 					        </li>
 			    		</ul>
 		    		</nav>
 	    		</div>
 	    	</div>
 	    	<!-- HEADER 2 -->
 
-	    	    <!-- reg and personal erea -->
-
-	    		<div class="nav-2 container-fluid d-flex justify-content-around">
-	    			<div class="left-buttons">
-		    			<button class="button-1">
-		    				<img src="img/icon-reg-for-website-mobil.png">
-		    			</button> 
-		    			<button class="button-2">
-		    				<img src="img/icon-reg-mobil.png">
-		    			</button> 
+    	    <!-- reg and personal erea -->
+    		<div class="nav-2 container-fluid d-flex align-items-center justify-content-between">
+	    		<div class="container d-flex justify-content-lg-between">
+	    			<div class="nav-icon-round-mobile hidden-lg-up">
+	    				<div class="mobile-menu d-flex align-items-center">
```

Comment:
`hidden-lg-up` not working for me when I look at the site on my computer.
Apparently the changed it... Let's talk on the phone and I will explain why but in any case you need to change the way we hide and show things for responsive view...

Example:
What we did until now to show element on mobile and hide on desktop was:
```
<div class="hidden-lg-up"></div>
```
You need to change it like this:
```
<div class="d-lg-none"></div>
```

The idea is you can say `d` (for display) then `xs`/`sm`/`md`/`lg`/`xl` or nothing followed by the display `none`/`block`/`flex` etc.


Some more examples:
```
<div class="d-flex d-lg-none"> // Display `flex` on all size up to `lg` and then `none`
<div class="d-none d-md-block"> // Display `none` on all size up to `md` and then `block` 
```